crates/blockifier/src/execution/contract_class.rs

```diff
@@ -432,6 +431,20 @@ fn convert_entry_points_v1(
         .collect()
 }
 
+#[cfg(feature = "parity-scale-codec")]
+fn hashmap_to_btree_v0(
+    hashmap: HashMap<EntryPointType, Vec<EntryPoint>>,
+) -> BTreeMap<EntryPointType, Vec<EntryPoint>> {
+    hashmap.into_iter().collect()
+}
+
+#[cfg(feature = "parity-scale-codec")]
+fn hashmap_to_btree_v1(
+    hashmap: HashMap<EntryPointType, Vec<EntryPointV1>>,
+) -> BTreeMap<EntryPointType, Vec<EntryPointV1>> {
+    hashmap.into_iter().collect()
+}
+
```

Comment:
Don't create two functions for this. Just write `hashmap.into_iter().collect()` directly in the original function body